fpga/src/main/scala/vcu118/HarnessBinders.scala

```diff
@@ -37,6 +40,32 @@ class WithDDRMem extends HarnessBinder({
   }
 })
 
+//Bare Metal Extension
+class WithVCU118UARTTSI extends HarnessBinder({
+  case (th: HasHarnessInstantiators, port: UARTTSIPort, chipId: Int) => {
+    val rawModule = th.asInstanceOf[LazyRawModuleImp].wrapper.asInstanceOf[VCU118FPGATestHarness]
+    val harnessIO = IO(new UARTPortIO(port.io.uartParams)).suggestName("uart_tsi")
+    harnessIO <> port.io.uart
+    val packagePinsWithPackageIOs = Seq(
+      ("AW25" , IOPin(harnessIO.rxd)),
+      ("BB21", IOPin(harnessIO.txd)))
```

Comment:
Your comment indicates that this overlaps with the UART device, which uses the UARTOverlay-generated pins in the VCU118TestHarness, but this generates its own pins. Do these overlap with the UARTOverlay pins?